src/main/java/org/truffleruby/language/supercall/LookupSuperMethodNode.java

```diff
@@ -44,15 +50,15 @@
     protected InternalMethod lookupSuperMethodCachedDynamicObject(VirtualFrame frame, DynamicObject self,
             @Cached("getCurrentMethod(frame)") InternalMethod currentMethod,
             @Cached("metaClass(self)") DynamicObject selfMetaClass,
-            @Cached("doLookup(currentMethod, selfMetaClass)") MethodLookupResult superMethod) {
+            @Cached("doLookup(frame, currentMethod, selfMetaClass)") MethodLookupResult superMethod) {
```

Comment:
AFAIU we should not change anything regarding @Cached. Correct me if I'm wrong, please